husarion_utils/rplidar.launch.py

```diff
@@ -0,0 +1,93 @@
+from launch import LaunchDescription
+from launch.actions import (
+    DeclareLaunchArgument,
+    IncludeLaunchDescription,
+    GroupAction,
+    OpaqueFunction,
+)
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros.actions import Node, PushRosNamespace, SetRemap
+from launch.substitutions import EnvironmentVariable, LaunchConfiguration, PathJoinSubstitution
+from launch_ros.substitutions import FindPackageShare
+
+
+def launch_setup(context, *args, **kwargs):
+    launch_file = LaunchConfiguration("launch_file").perform(context)
+    serial_baudrate = LaunchConfiguration("serial_baudrate").perform(context)
+    serial_port = LaunchConfiguration("serial_port").perform(context)
+    robot_namespace = LaunchConfiguration("robot_namespace").perform(context)
+    device_namespace = LaunchConfiguration("device_namespace").perform(context)
+
+    if device_namespace:
+        frame_id = device_namespace + "_link"
+    else:
+        frame_id = "scan"
```

Comment:
As default I think it should be `laser` as a `frame_id`.
https://github.com/husarion/ros_components_description/blob/ros2/urdf/slamtec_rplidar_s1.urdf.xacro#L8C24-L8C32
```suggestion
        frame_id = "laser"
```